doc/requirements/eviction

```diff
@@ -0,0 +1,27 @@
+---
+group: Eviction
+---
+
+Eviction controlls a process of invalidating clean data from cache.
```

Comment:
There are other ways to invalidate cachelines as well. I would say that "Eviction is a process of invalidating cachelines in order to map to a new addresses"